src/csvcubed/readers/cubeconfig/utils.py

```diff
@@ -35,17 +36,20 @@ def generate_title_from_file_name(csv_path: Path) -> str:
     )
 
 
-def read_and_check_csv(csv_path: Path) -> Tuple[DataFrame, List[ValidationError]]:
+def read_and_check_csv(
+    csv_path: Path, dtype: Optional[Dict[str, str]] = None
+) -> Tuple[DataFrame, List[ValidationError]]:
     """
     Reads the csv data file and performs rudimentary checks.
     """
-    data, data_errors = read_csv(csv_path)
+
+    data, data_errors = read_csv(csv_path, dtype=dtype)
 
     if isinstance(data, DataFrame):
-        if data.shape[0] < 2:
+        if len(data) == 0:
```

Comment:
I guess this'll fix that bug we have where it complains if you only have a header row and a single data row!